benchmarks/src/main/java/io/grpc/benchmarks/ByteBufOutputMarshaller.java

```diff
@@ -40,7 +40,7 @@
 import java.io.InputStream;
 
 /**
- * Simple {@link MethodDescriptor#Marshaller} for Netty's {@link ByteBuf}.
+ * Simple {@link io.grpc.MethodDescriptor.Marshaller} for Netty's {@link ByteBuf}.
```

Comment:
Just yellow squiggles.  I'm not sure the intersection of IDEs that people use are smart enough to resolve this ref.  Not that this change make it any worse.